arcgis-ios-sdk-samples/Utility Network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -15,45 +15,60 @@
 import UIKit
 import ArcGIS
 
-private let barrierAttributeValue = "barrier"
-
-class ConnectedTraceViewController: UIViewController, AGSGeoViewTouchDelegate {
+class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelegate {
     @IBOutlet weak var mapView: AGSMapView!
 
-    @IBOutlet weak var traceNetworkButton: UIButton!
-    @IBOutlet weak var resetButton: UIButton!
+    @IBOutlet weak var traceNetworkButton: UIBarButtonItem!
+    @IBOutlet weak var resetButton: UIBarButtonItem!
+    @IBOutlet weak var typeButton: UIBarButtonItem!
     @IBOutlet weak var modeLabel: UILabel!
-    @IBOutlet weak var modeControl: UISegmentedControl!
     @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet weak var modeControl: UISegmentedControl!
     
     private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
+    
+    // Create electrical distribution line layer ./115 and electrical device layer ./100.
     private var layers: [AGSFeatureLayer] {
         return [115, 100].map {
             let featureTable = AGSServiceFeatureTable(url: featureServiceURL.appendingPathComponent("\($0)"))
             let layer = AGSFeatureLayer(featureTable: featureTable)
             if $0 == 115 {
-                let lineColor = UIColor(red: 0, green: 0.55, blue: 0.55, alpha: 1)
-                layer.renderer = AGSSimpleRenderer(symbol: AGSSimpleLineSymbol(style: .solid, color: lineColor, width: 3))
+                // Define a solid line for medium voltage lines and a dashed line for low voltage lines.
+                let darkCyan = UIColor(red: 0, green: 0.55, blue: 0.55, alpha: 1)
+                let mediumVoltageValue = AGSUniqueValue(description: "N/A",
+                                                        label: "Medium voltage",
+                                                        symbol: AGSSimpleLineSymbol(style: .solid, color: darkCyan, width: 3),
+                                                        values: [5])
+                let lowVoltageValue = AGSUniqueValue(description: "N/A",
+                                                     label: "Low voltage",
+                                                     symbol: AGSSimpleLineSymbol(style: .dash, color: darkCyan, width: 3),
+                                                     values: [3])
+                layer.renderer = AGSUniqueValueRenderer(fieldNames: ["ASSETGROUP"],
+                                                        uniqueValues: [mediumVoltageValue, lowVoltageValue],
+                                                        defaultLabel: "",
+                                                        defaultSymbol: AGSSimpleLineSymbol()
+                )
```

Comment:
I know having things pushed so far to the right is rather common in Objective-C, but in Swift, we usually do things a little differently:

```suggestion
                let mediumVoltageValue = AGSUniqueValue(
                    description: "N/A",
                    label: "Medium voltage",
                    symbol: AGSSimpleLineSymbol(style: .solid, color: darkCyan, width: 3),
                    values: [5]
                )
                let lowVoltageValue = AGSUniqueValue(
                    description: "N/A",
                    label: "Low voltage",
                    symbol: AGSSimpleLineSymbol(style: .dash, color: darkCyan, width: 3),
                    values: [3]
                )
                layer.renderer = AGSUniqueValueRenderer(
                    fieldNames: ["ASSETGROUP"],
                    uniqueValues: [mediumVoltageValue, lowVoltageValue],
                    defaultLabel: "",
                    defaultSymbol: AGSSimpleLineSymbol()
                )
```

Reply:
Right! I see it pretty awkwardly aligned, but forget where I should cut off the first line 😅 and auto re-indent tool does not fix it.